modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -62,8 +70,12 @@ $ oc apply -f csc.cr.yaml
 ----
 
 . Create a Subscription object YAML file (for example, `myoperator-sub.yaml`) to
-subscribe a namespace to an Operator:
+subscribe a namespace to an Operator. Note that the namespace you pick
+must have an OperatorGroup that matches the installMode (either AllNamespaces or
+SingleNamespace modes):
 +
+.Example Subscription
+[source,yaml]
 ----
 apiVersion: operators.coreos.com/v1alpha1
```

Comment:
The example Subscription is missing the `spec.sourceNamespace` field under `spec`:
```
kind: Subscription
metadata:
  name: myoperator
  namespace: openshift-operators
spec:
  channel: alpha
  name: myoperator <1>
  source: example <2>
  sourceNamespace: openshift-operators
```

Reply:
Okay. Thanks!